# FlatInput
#
# Omar Elmofty 2020-03-10 -- For use in AER1217 Winter 2017
#
# Data communicates the flat input of the quadrotor
# following order [x_dddot, y_dddot, z_dddot, yaw_dot]

float64[4] flat_input
